### PR TITLE
formatted lerna completion and used helper function

### PR DESCRIPTION
**Pull request by NoahGorny**
Cleanup lerna completion as part of #1782
Untested

### CODE REVIEW

File: completion/available/lerna.completion.bash [hunk 1]
```diff
@@ -1,19 +1,23 @@
-#!/bin/bash
+# shellcheck shell=bash
 # Lerna autocompletion.
-function _lerna_completions() {
-  local cur compls
 
-  # The currently-being-completed word.
-  cur="${COMP_WORDS[COMP_CWORD]}"
+if _command_exists lerna; then
+	function _lerna_completions() {
```
Comment (cornfeedhobo) [inline completion/available/lerna.completion.bash:5]: Could we drop the plural and use double underscore prefix?

`__lerna_completion`

File: completion/available/lerna.completion.bash [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
+        import init link list publish run version    \
+        --loglevel --concurrency --reject-cycles    \
+        --progress --sort --no-sort --help          \
+        --version"
+
+		# Tell complete what stuff to show.
+		# shellcheck disable=2207
+		COMPREPLY=($(compgen -W "$compls" -- "$cur"))
+	}
+	complete -o default -F _lerna_completions lerna
```
Comment (cornfeedhobo) [inline completion/available/lerna.completion.bash:22]: same as above

File: completion/available/lerna.completion.bash (hunk 1 above)
Comment (NoahGorny) [reply completion/available/lerna.completion.bash]: fixed

Comment (cornfeedhobo) [reply completion/available/lerna.completion.bash]: @NoahGorny almost. looks like it still only uses a single underscore prefix

**Review by cornfeedhobo — APPROVED** [review]

**Pull request merged**